classes/navigationMenu/NavigationMenuItemDAO.php

```diff
@@ -111,15 +112,15 @@ public function getByTypeAndTitleLocaleKey(int $contextId, string $menuItemType,
      *
      * @return DAOResultFactory<NavigationMenuItem>
      */
-    public function getByType(string $type, ?int $contextId = null): DAOResultFactory
+    public function getByType(string $type, ?int $contextId = Application::SITE_CONTEXT_ID_ALL)
```

Comment:
This removes the return typehint; is that intentional?

Reply:
I didn't notice this one while rebasing/merging 😁